webapp-django/crashstats/settings/bundles.py

```diff
@@ -0,0 +1,468 @@
+        'source_filenames': (
+            'manage/js/fields.js',
+        ),
+        'output_filename': 'js/manage-fields.min.js',
+    },
+    'manage:graphics_devices': {
+        'source_filenames': (
+            'manage/js/graphics_devices.js',
+        ),
+        'output_filename': 'js/manage-graphics-devices.min.js',
+    },
+    'manage:groups': {
+        'source_filenames': (
+            'manage/js/groups.js',
+        ),
+        'output_filename': 'js/manage-groups.min.js',
+    },
+    'manage:skiplist': {
+        'source_filenames': (
+            'manage/js/skiplist.js',
+        ),
+        'output_filename': 'js/manage-skiplist.min.js',
+    },
+    'manage:supersearch_field': {
+        'source_filenames': (
+            'manage/js/supersearch_field.js',
+        ),
+        'output_filename': 'js/manage-supersearch-field.min.js',
+    },
+    'manage:supersearch_fields': {
+        'source_filenames': (
+            'manage/js/supersearch_fields.js',
+        ),
+        'output_filename': 'js/manage-supersearch-fields.min.js',
+    },
+    'manage:symbols_uploads': {
+        'source_filenames': (
+            'manage/js/symbols-uploads.js',
+        ),
+        'output_filename': 'js/manage-symbols-uploads.min.js',
+    },
+    'manage:users': {
+        'source_filenames': (
+            'manage/js/users.js',
+        ),
+        'output_filename': 'js/manage-users.min.js',
+    },
+    'profile': {
+        'source_filenames': (
+            'profile/js/profile.js',
+        ),
+        'output_filename': 'js/profile.min.js',
+    },
+    'signature_report': {
+        'source_filenames': (
+            'signature/js/signature_report.js',
+            'signature/js/signature_tab.js',
+            'signature/js/signature_tab_summary.js',
+            'signature/js/signature_tab_graphs.js',
+            'signature/js/signature_tab_reports.js',
+            'signature/js/signature_tab_aggregations.js',
+            'signature/js/signature_tab_comments.js',
+            'signature/js/signature_tab_graph.js',
+            'signature/js/signature_panel.js',
+        ),
+        'output_filename': 'js/signature-report.min.js',
+    },
+    'search_custom': {
+        'source_filenames': (
+            'supersearch/js/lib/ace/ace.js',
+            'supersearch/js/lib/ace/theme-monokai.js',
+            'supersearch/js/lib/ace/mode-json.js',
+            'supersearch/js/socorro/search_custom.js',
+        ),
+        'output_filename': 'js/search-custom.min.js',
+    },
+    'search': {
+        'source_filenames': (
+            'supersearch/js/socorro/search.js',
+        ),
+        'output_filename': 'js/search.min.js',
+    },
+    'tokens': {
+        'source_filenames': (
+            'tokens/js/home.js',
+        ),
+        'output_filename': 'js/tokens.min.js',
+    },
+}
+
+# This is sanity checks, primarily for developers. It checks that
+# you haven't haven't accidentally make a string a tuple with an
```

Comment:
"you haven't <strike>haven't</strike> accidentally <strike>make</strike>made a string a tuple with an"